beanstalk_worker/worker_start_build.py

```diff
@@ -48,20 +48,13 @@ def run_command(command):
         logger.log(level=logging.CRITICAL, msg=e.message)
         return e.message
 
-def notify_build_failure(namespace, notify_email, log_command):
+def notify_build_failure(namespace, notify_email, build_logs_file):
     msg_details = {}
     msg_details['action'] = 'notify_user'
-    msg_details['subject'] = "FAILED: Container-build failed " + namespace
```

Comment:
We're removing `subject` part from the message here. Are we taking care of it somewhere else? Meaning, will the email being sent out to the user have a subject?